src/components/resources/Sidebar/SidebarItem.tsx

```diff
@@ -39,14 +57,14 @@ export default function SidebarItem(props: SidebarItemProps) {
           {/* if you have children, display a dropdown arrow */}
           {props.item.children ? (
             <button className="menu-down" onClick={() => setOpen(!open)}>
-              <i className={`fa fa-chevron-${open ? "up" : "down"}`} />
+              <i className={`fa fa-chevron-${open ? "up" : "down"}`} />               
             </button>
           ) : null}
         </div>
       </div>
 
       {/* if you have children, display children */}
-      {props.item.children ? (
+      {props.item.children && (
```

Comment:
There is a weird intentional practice here: https://medium.com/geekculture/stop-using-for-conditional-rendering-in-react-a0f7b96200f8

Reply:
resolved: [53826fd](https://github.com/CarletonComputerScienceSociety/resources/pull/6/commits/53826fd65f51b579b7001c4b65a674fec8cdb109)